### PR TITLE
WIP: ENS Tenant cleanup 

### PR DESCRIPTION
**Pull request by andreitr**
- fixed duplicate preview OG images
- added ens and etherfi logos to OG previews
- moved links (calendar, changelog, etc) into the tenant
- a few misc improvements

---

## PR-Codex overview
The focus of this PR is to update metadata handling, add links to UI, and fix contract imports.

### Detailed summary
- Updated metadata handling for pages
- Added links to UI for calendar, FAQ, changelog, Discord, bugs
- Fixed contract imports for better organization

> The following files were skipped due to too many changes: `src/assets/tenant/optimism_logo.svg`, `src/app/api/common/governanceCalendar/getGovernanceCalendar.ts`, `src/assets/tenant/ens_logo.svg`, `src/assets/tenant/etherfi_logo.svg`, `src/app/api/images/og/assets/shared.tsx`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: src/app/api/common/governanceCalendar/getGovernanceCalendar.ts [hunk 1]
```diff
@@ -1,104 +1,125 @@
...
-    const response = await fetch(
-        `https://calendar.google.com/calendar/ical/c_fnmtguh6noo6qgbni2gperid4k%40group.calendar.google.com/public/basic.ics`,
-        {
-            method: "GET"
-        }
-    );
-    const calendarICS = await response.text();
-    return parseICS(calendarICS);
+  const { ui } = Tenant.current();
+  const link = ui.link("calendar");
```
Comment (andreitr) [inline src/app/api/common/governanceCalendar/getGovernanceCalendar.ts:114]: This is the main change, the rest is prettier

File: src/lib/tenant/tenantContractFactory.ts [hunk 2]
```diff
@@ -34,6 +35,39 @@ export default class TenantContractFactory {
...
+    // TOKEN
+    token: new TenantContract<ITokenContract>({
+      contract: EtherfiToken__factory.connect(
+        "0xC18360217D8F7Ab5e7c516566761Ea12Ce7F9D72",
+        ethProvider
+      ),
+      address: "0xC18360217D8F7Ab5e7c516566761Ea12Ce7F9D72" as `0x${string}`,
+      chainId: 1,
+      chainName: "Ethereum Mainnet",
+      abi: EtherfiToken__factory.abi,
```
Comment (stepandel) [inline src/lib/tenant/tenantContractFactory.ts:49]: Is this supposed to be `ENSToken_factory`?  or it's generic ERC-20 contract?

File: src/lib/tenant/tenantContractFactory.ts [hunk 3]
```diff
@@ -34,6 +35,39 @@ export default class TenantContractFactory {
...
+    }),
+    // GOVERNOR
+    // TODO: Implement Sepolia provider
+    governor: new TenantContract<IGovernorContract>({
+      contract: OptimismGovernor__factory.connect(
```
Comment (stepandel) [inline src/lib/tenant/tenantContractFactory.ts:54]: fyi here as well

File: src/lib/tenant/tenantContractFactory.ts (hunk 2 above)
Comment (andreitr) [reply src/lib/tenant/tenantContractFactory.ts:49]: @stepandel, that's a temporary placeholder. For ENS, Etherfi, and all other ERC20 tokens, I plan to create an interchangeable class. This works for now but I will make a change this week.

**Review by ferrodri — APPROVED** [review]
LGTM :)

**Pull request merged**